vpr/src/route/connection_box_lookahead_map.h

```diff
@@ -32,63 +32,46 @@ struct RoutingCostKey {
     }
 };
 
-// compressed version of RoutingCostKey
-// TODO add bounds checks
-struct CompressedRoutingCostKey {
-    uint32_t data;
+// Data in the RoutingCosts map
+struct RoutingCost {
+    // source and destination node indices
+    int from_node, to_node;
```

Comment:
I don't believe either of these fields actually get used?

Reply:
The struct isn't used at all, so I removed it.